numba/tests/test_tuples.py

```diff
@@ -745,8 +745,8 @@ def foo():
 
         with self.assertRaises(errors.UnsupportedError) as raises:
             foo()
-
-        msg = "op_LIST_EXTEND at the start of a block"
+        if utils.PYVERSION >= (3, 9):
+            msg = "op_LIST_EXTEND at the start of a block"
         self.assertIn(msg, str(raises.exception))
```

Comment:
The test is already skipped on py<3.9. The `if` is unnecessary. Also, `msg` is only defined in the `if` block. but used after the `if`.

Reply:
done in 9b0e74f86eb8e63c966f362a130607f3cc97e612